examples/autograd/singa-onnx/onnx_model.py

```diff
@@ -0,0 +1,43 @@
+import pickle
```

Comment:
pls add the license header to every file you added

https://github.com/apache/incubator-singa/blob/master/examples/autograd/mlp.py#L1

Reply:
yes